build.gradle

```diff
@@ -44,20 +44,8 @@ nexusStaging {
     delayBetweenRetriesInMillis = 5000
 }
 
-// Required for Javadoc compilation
-apply plugin: "org.openjfx.javafxplugin"
-
-repositories {
-    mavenCentral()
-}
-
-javafx {
-    modules 'javafx.graphics'
```

Comment:
Why do you remove this? It is a required dependency for almost every subproject (in fact you abstracted it)?

Reply:
The block where is gets added to all subprojects still [exists in the subprojects block](https://github.com/gluonhq/attach/blob/d4cc0664436a7ace7aace8436b688c09f3c336a0/build.gradle#L34).

The one I have removed was only added to the root module to compile Javadoc.

Reply:
Ah, right.